### PR TITLE
code-review done

### PR DESCRIPTION
**Pull request by Shaybush**

### CODE REVIEW

**Comment by Shaybush** [comment]
@avivt10 I added some comments for changes. all the comments start // TODO -

File: src/components/header/search/search.tsx [hunk 1]
```diff
@@ -6,16 +6,24 @@ import { currentSelectCountry } from "../../../redux/features/currentCountrySlic
...
+  // TODO - why don't use form ? if you are using button as a type=submit then handleClickSearch trigger. 
     const inputRef = useRef<HTMLInputElement>(null);
       const dispatch = useAppDispatch();  
 
     const handleClickSearch = async () => {
         if (inputRef.current) {
             let searchTerm = inputRef.current.value;
+            // TODO - why do need to capitalize ? 
+            // https://disease.sh/v3/covid-19/countries/israel - works for me as lower case
+            // you can just trim and make as a lowercase --> searchTerm.trim().toLowerCase()
```
Comment (avivt10) [inline src/components/header/search/search.tsx:22]: I want the first letter to be capitalized in header --> israel = Israel

File: src/components/header/search/search.tsx [hunk 2]
```diff
@@ -27,6 +35,7 @@ const Search = () => {
     };
   return (
     <div className="searchContainer">
+      {/* TODO - why not using <form onSubmit={handleClickSearch}>  */}
```
Comment (avivt10) [inline src/components/header/search/search.tsx:38]: done.

File: src/components/main/main.tsx [hunk 3]
```diff
@@ -20,17 +53,32 @@ const Main = () => {
       {currentCountry !== "" ? (
         <div className="mainContainer">
           <img
+            // TODO - move /src/assets/image/covid.png to variable, like: IMG_SOURCE = "/src/assets/image/covid.png"
             src="/src/assets/image/covid.png"
             className="covidImgSpin"
+            // TODO - width and height should place in variable IMG_HEIGHT=50
```
Comment (avivt10) [inline src/components/main/main.tsx:59]: why?

File: src/redux/features/dataCountrySlice.ts [hunk 4]
```diff
@@ -5,6 +5,8 @@ interface dataCountryState {
     dataCountry: dataTypeModel;
 }
 
+// TODO - why do you need to store all that data ???
+// which props are u using ? remove all the unnecessary variables please
```
Comment (avivt10) [inline src/redux/features/dataCountrySlice.ts:9]: done.

File: src/redux/models/dataTypeModel.ts [hunk 5]
```diff
@@ -1,3 +1,4 @@
+// TODO - Do you need all that types ?
```
Comment (avivt10) [inline src/redux/models/dataTypeModel.ts:1]: done.

File: src/shared/components/StatisticsCircle/StatisticsCircle.tsx [hunk 6]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+// TODO - comments just in english, delete that or translate
```
Comment (avivt10) [inline src/shared/components/StatisticsCircle/StatisticsCircle.tsx:3]: done.

File: src/components/header/header.tsx [hunk 7]
```diff
@@ -3,6 +3,7 @@ import Search from "./search/search";
 
 const Header = () => {
   return (
+    // TODO - why do you need empty tags here
```
Comment (avivt10) [inline src/components/header/header.tsx:6]: done.

File: src/components/header/search/search.tsx [hunk 8]
```diff
@@ -6,16 +6,24 @@ import { currentSelectCountry } from "../../../redux/features/currentCountrySlic
...
     const handleClickSearch = async () => {
         if (inputRef.current) {
             let searchTerm = inputRef.current.value;
+            // TODO - why do need to capitalize ? 
+            // https://disease.sh/v3/covid-19/countries/israel - works for me as lower case
+            // you can just trim and make as a lowercase --> searchTerm.trim().toLowerCase()
             searchTerm = searchTerm.charAt(0).toUpperCase() + searchTerm.slice(1);
 
             try {
+                // TODO - usually domain url sit on .env file BASE_URL=https://disease.sh/v3/covid-19
```
Comment (avivt10) [inline src/components/header/search/search.tsx:26]: done.

**Pull request closed without merging**